elm-package.json

```diff
@@ -1,28 +1,26 @@
 {
-    "version": "0.2.0",
+    "version": "1.0.0",
     "summary": "feed the tree some ether.",
     "repository": "https://github.com/cmditch/elm-web3.git",
     "license": "BSD3",
     "source-directories": [
-        "src"
-    ],
-    "exposed-modules": [
-        "Web3",
```

Comment:
Any reason to delete the exposed modules?

Reply:
Yeah - will re-merge master in and see if that resolves.